packages/rocket-batch-file-process-aws-infrastructure/README.md

```diff
@@ -32,9 +38,16 @@ Booster.configure('development', (config: BoosterConfig): void => {
   config.provider = Provider([{
     packageName: '@boostercloud/rocket-batch-file-process-aws-infrastructure', 
     parameters: {
-      bucketName: 'test-bucket-name', 
-      chunkSize: '2',
-    }
+      config: {
+        bucketName: 'test-bucket',
+        chunkSize: '2',
+      },
+      rowEvent: {
+        entityId: 'id',
+        eventTypeName: 'RowAdded',
```

Comment:
When developing rockets, we have to be aware of too standard names, because it's easy that they collide with existing or future application event names. Ideally we should find a way to "scope" our rocket events, like using `BatchProcessor.RowAdded` instead of just `RowAdded`. Or an alternative If that's not possible, you get the point.

Reply:
@javiertoledo Actually the `RowAdded` comes from the Booster app and is defined by the user. I think my README is definitely not clear. I will improve it :) 

This is the example of the Booster app.